### PR TITLE
Added missing Ps command

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/816

### CODE REVIEW

**Comment by mypil** [comment]
@KCCross - PR is ready for final review.  

Thank you.

cc: @kenwith

**Review by samschan-ms — APPROVED** [review]

**Comment by cabailey** [comment]
Ready to merge after PM approval

**Pull request merged**
